webdriver/tests/bidi/network/continue_response/invalid.py

```diff
@@ -0,0 +1,98 @@
+    with pytest.raises(error.NoSuchRequestException):
+        await bidi_session.network.continue_response(request=value)
+
+
+async def test_params_request_no_such_request(bidi_session, setup_network_test,
+                                              wait_for_event, fetch, url):
+    await setup_network_test(events=[
+        "network.responseCompleted",
+    ])
+    on_response_completed = wait_for_event("network.responseCompleted")
+
+    text_url = url(PAGE_EMPTY_TEXT)
+    await fetch(text_url)
+
+    response_completed_event = await on_response_completed
+    request = response_completed_event["request"]["request"]
+
+    with pytest.raises(error.NoSuchRequestException):
+        await bidi_session.network.continue_response(request=request)
+
+
+@pytest.mark.parametrize("value", [False, 42, {}, []])
+async def test_params_reason_phrase_invalid_type(bidi_session, value,
+                                                 setup_network_test, url, fetch,
+                                                 wait_for_event, add_intercept):
+    request = await create_blocked_request(setup_network_test, url,
+                                           add_intercept, fetch,
+                                           wait_for_event)
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.network.continue_response(request=request,
+                                                     reason_phrase=value)
+
+
+@pytest.mark.parametrize("value", [False, "s", {}, []])
+async def test_params_status_code_invalid_type(bidi_session, value, setup_network_test,
+                                               url, fetch, wait_for_event,
+                                               add_intercept):
+    request = await create_blocked_request(setup_network_test, url,
+                                           add_intercept, fetch,
+                                           wait_for_event)
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.network.continue_response(request=request, status_code=value)
+
+
+@pytest.mark.parametrize("value", [-1, 4.3])
+async def test_params_status_code_invalid_value(bidi_session, value, setup_network_test,
+                                                url, fetch, wait_for_event,
+                                                add_intercept):
+    request = await create_blocked_request(setup_network_test, url,
+                                           add_intercept, fetch,
+                                           wait_for_event)
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.network.continue_response(request=request, status_code=value)
+
+# TODO: Add cookies.
+# TODO: Add credentials.
+# TODO: Add headers.
+
+
+async def create_blocked_request(setup_network_test, url, add_intercept, fetch,
+                                 wait_for_event):
+    await setup_network_test(events=["network.beforeRequestSent"])
```

Comment:
We should listen to responseStarted here, otherwise the intercept will not be triggered.

note: This probably means the tests are passing because the request is done and that's what's causing the invalid argument exception. It's too bad we can't assert this in more details

Reply:
Do you know if "responseCompleted" would be OK as well?

Reply:
Done.